src/components/Common/sampleItems.ts

```diff
@@ -39,6 +39,26 @@ export const sampleItems = [
   },
   {
     index: 2,
+    title: "Canister HTTP Calls",
+    image: require("../../../static/img/samples/exchangeRate.png").default,
+    domains: ["HTTP", "Exchange", "Global", "Website"],
+    languages: ["Rust", "Javascript"],
+    level: ["Intermediate"],
+    contentType: ["Documentation"],
+    highlights: ["HTTP", "Exchange", "Website", "Global", "Intermediate"],
+    body: "Demonstrates how to build a dapp with canister outgoing HTTP calls.",
+    links: {
+      action: {
+        text: "Get Code",
+        to: "https://github.com/dfinity/examples/tree/master/rust/exchange_rate",
+      },
+      docs: "docs/current/samples/http-requests-exchange-rates",
+      rust: "https://github.com/dfinity/examples/tree/master/rust/exchange_rate",
+      livePreview: "https://mfwph-oqaaa-aaaam-qafsq-cai.ic0.app/",
```

Comment:
This doesn't seem like it points to the right thing. Looks to me like this links to a cycles wallet